src/main/java/uk/gov/hmcts/reform/cdapi/service/impl/CrdServiceImpl.java

```diff
@@ -95,6 +97,9 @@ private List<Category> convertCategoryList(List<ListOfValueDto> listOfValueDtos)
                 .valueEn(dto.getValueEn())
                 .parentCategory(dto.getParentCategory())
                 .parentKey(dto.getParentKey())
-                .build()).collect(Collectors.toUnmodifiableList());
+                .build())
```

Comment:
if LovOrder value is not provided, then we have to sort by Name.

Reply:
sorting by valueEn, as discussed